model/atmosphere/dycore/tests/dycore_stencil_tests/test_solve_tridiagonal_matrix_for_w_back_substitution.py

```diff
@@ -23,7 +26,12 @@ class TestSolveTridiagonalMatrixForWBackSubstitution(StencilTest):
     OUTPUTS = ("w",)
 
     @staticmethod
-    def reference(grid, z_q: np.array, w: np.array, **kwargs) -> dict:
+    def reference(
+        connectivities: dict[gtx.Dimension, np.ndarray],
+        z_q: np.ndarray,
+        w: np.ndarray,
+        **kwargs: Any,
+    ) -> dict:
```

Comment:
Just a question (which applies to all files): shouldn't the return type of the reference method return something like `dict[str, Field | scalar]`?

Reply:
I think that is correct. Don't know whether I want to touch it again though...